### ISSUE TITLE
carddav fails in roundcube 1.4rc2

### ISSUE THREAD

**Issue by henrythemouse** [blind-coder/rcmcarddav#267]
I just recently updated to the 'complete' versoin of v1.4rc2  from rc1. My custom addressbooks disappeared in v1.4rc2. I'm using davical which is working fine when I access my addresses from infCloud.

I get this php error in errors.log (no errors in carddav logs) when I try to access the carddav settings:

`PHP Fatal error: Uncaught Error: Call to undefined function Sabre\HTTP\curl_init()`

I have current versions of php-curl  and curl installed. When I try to create a new carddav addressbook in rc the form section goes blank white. I can reload the form again but nothing is saved in the carddav related db tables (except in the 'carddav_migrations' table). And the input form doesn't show any saved carddav settings. 

I'd appreciate any help you can offer. 
TIA

[carddavError.txt](https://github.com/blind-coder/rcmcarddav/files/3714438/carddavError.txt)

**Comment by blind-coder** [comment]
That sounds like the SabreDAV library is no longer installed. Please 
reinstall RCMCardDAV as per the installation instructions, depending on 
the way you installed it.

-- 
Jabber: benjamin@anderdonau.de
Twitter: https://twitter.com/blind_coder
Web: https://www.benjamin-schieder.de/

**Comment by henrythemouse** [comment]
You got me thinking about things when you mentioned possible missing carddav files, that didn't seem likely. Around the time this problem started I upgraded my system to debian Buster. The upgrade got interrupted by 'life' and I had some issues completing it when I got back to it. That aside, the problem here was that php7.3 was never fully installed and apache2 was using a nonfunctioning installation of 7.0. Once I installed  libapache2-mod-php7.3 and enabled it in apache2 the carddav functionality returned. Makes me wonder what else I messed up on when I did that upgrade.

Thanks for sparking that idea, leading me to solving this problem.
